File: pneumatics.py
# configure pneumatics
pneumatics = DigitalOut(brain.three_wire_port.a)

# set cylinder to down
digital_out_a.set(False)

if controller.buttonL1.pressing():
    # cylinder up
    digital_out_a.set(True)
elif controller.buttonL2.pressing():
    # cylinder down
    digital_out_a.set(False)

File: pneumatics.cpp
// Configure pneumatics
digital_out pneumatics = digital_out(Brain.ThreeWirePort.A);

// Set cylinder to down
pneumatics.set(false);

if (Controller1.ButtonL1.pressing()) {
    // Cylinder up
    pneumatics.set(true);
}
else if (Controller1.ButtonL2.pressing()) {
    // Cylinder down
    pneumatics.set(false);
}
